apps/authorization/management/commands/update_access_grants.py

```diff
@@ -1,27 +1,35 @@
 from django.core.management.base import BaseCommand
 from apps.authorization.models import DataAccessGrant
 from apps.dot_ext.models import Application
+
+
 class Command(BaseCommand):
     help = (
         'Update Access Grants Per Application.'
         'Pass in a list of application ids.'
     )
 
     def add_arguments(self, parser):
-        parser.add_argument('--applications', help="list of applications to update "
+        parser.add_argument('--applications', help="List of applications to update "
                                                     "id, comma separated: "
-                                                    "eg. 1,2,3 ")
+                                                    "eg. 1,2,3. Supersedes --all")
```

Comment:
Instead of superseding `--all`, it might be clearer to have the two options be mutually exclusive. Optional, just an idea!

Reply:
If we were introducing outside our team I probably would, lol. I may still! I thought that for us and future internal teams, the help string would make it make sense and defaulting to the less destructive option would at least limit accidents.